scripts/box

```diff
@@ -227,42 +228,44 @@ function _adduser() {
         exit 1
     fi
 
-    if [[ $(cut -d: -f1 < /root/.master.info 2> /dev/null) = "$user" ]] && [[ -z $SETUP_USER ]]; then
-        echo_error "User already exists within swizzin"
-        exit 1
+    if [[ -z $SETUP_USER ]]; then #If this is not coming from setup.sh...
+        if [[ "$(cut -d: -f1 < /root/.master.info 2> /dev/null)" = "$user" ]]; then
+            echo_error "User already exists within swizzin"
+            exit 1
+        fi
     fi
 
     if id -u "${user}" > /dev/null 2>&1; then
-        echo_info "The user ${user} already appears to be present on this machine; however, it does not appear to be configured as a swizzin user"
-        ask "Continue setting up user?" N || exit 1
+        echo_info "The user ${user} already appears to be present on this machine; however, it does not appear to be configured as a swizzin user.\nThe user will be added to swizzin."
+        ask "Continue setting up user?" Y || exit 1
```

Comment:
Should there even be confirmation logic here?

Reply:
I think we should keep it for now as I think users should be aware that there may be quirks if they have already setup a user